spec/lib/stripe/authorize_response_patcher_spec.rb

```diff
@@ -37,7 +37,7 @@
             "status" => "requires_source_action",
             "next_source_action" => {
               "type" => "authorize_with_url",
-              "authorize_with_url" => { "url" => "https://www.stripe.com.malicious.org/authorize" }
+              "authorize_with_url" => { "url" => "https://www.evil-stripe.com.malicious.org/authorize" }
```

Comment:
I think this makes the test less likely to catch a trick (it doesn't test the slightly more innocent-looking urls `www.evil-stripe.com/` or `www.stripe.com.malicious.org/`)
But.. at least it's checking that the validation is doing something, which is probably enough in this case.